UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -104,6 +112,18 @@ private void OnControllerColliderHit(ControllerColliderHit hit)
     {
         bool isMovingUpwards = verticalMovement > 0f;
 
+        // Calculates if hit vector is above slopelimit set on Character Controller
+        isAboveSlopeLimit = (Vector3.Angle (Vector3.up, hit.normal) >= characterController.slopeLimit);
+        slideMovement = new Vector3(hit.normal.x, hit.normal.y, hit.normal.z);
+
+        if (isAboveSlopeLimit) 
```

Comment:
can shorten it to

isSliding = isAboveSlopeLimit

Reply:
You're right. I just changed it. Thanks :)